hardhat.config.ts

```diff
@@ -15,6 +15,18 @@ const config: HardhatUserConfig = {
           },
         },
       },
+      {
+        // 0.8.19 compiler is only used for the VotingTokenLockupPlans contract,
```

Comment:
To make the compiler version explicit for a single contract only, we could use [solidity.overrides](https://hardhat.org/hardhat-runner/docs/advanced/multiple-solidity-versions):

```
solidity: {
  compilers: [...],
  overrides: {
      "contracts/mocks/hedgey/VotingTokenLockupPlans.sol": {
        version: '0.8.19',
        settings: {
          optimizer: {
            enabled: true,
            runs: 200,
          },
          viaIR: true,
        },
      }
    }
```